packages/locky/openapi3.yaml

```diff
@@ -0,0 +1,148 @@
+openapi: 3.0.1
+info:
+  title: locky
+  description: api for managing locking services in the vector ecosystem
+  version: 1.0.0
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+tags:
+  - name: lock
+    description: a lock on service(s)
+paths:
+  /lock:
+    post:
+      operationId: postLock
+      tags:
+        - lock
+      summary: create a lock on services
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/lockRequest'
+      responses:
+        201:
```

Comment:
This is not a valid JSON, it is a string.

Reply:
done.